virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,47 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to a Linux bridge network
```

Comment:
Bob, I'm trying to recall what Jean-Francois said. Didn't he say that he preferred "additional" network? Jira seems to be 'down' right now, so I can't check the comments.

Reply:
He didn't give a preference so I think we should keep consistency and stick with Linux bridge network. JF gave it a LGTM.